docs/earth/forecast.md

```diff
@@ -8,3 +8,146 @@ kernelspec:
 ---
 
 # Weather Forecasts
+
+One way to model forecast output is a datacube with four dimensions :
+
+- 2 spatial dimensions `x` and `y`,
+- a forecast reference time (or initialization time) commonly `time`[datetime64], and
+- a forecast period or "timesteps in to the future" commonly `step`[timedelta64]
+
+There is one ancillary variable: 'valid time' which is the time for which the forecast is made: `valid_time = time + step`
+
+```{note}
+Note that not all forecast model runs are run for the same length of time! We could model these missing forecasts as NaN values in the output.
+```
+
+There are many ways one might index weather forecast output.
+These different ways of constructing views of a forecast data are called "Forecast Model Run Collections" (FMRC).
+For reference, see [this classic image](https://www.unidata.ucar.edu/presentations/caron/FmrcPoster.pdf) where the
+y-axis is the 'valid time', and the x-axis is the 'forecast reference or initialization time':
+![FMRC indexing schematic](./fmrc.png)
```

Comment:
I definitely am having to stare at this for a while to make sense of it :) I think I'm distracted by the too-small-to-read annotation bboxes in the PNG... Could mention the link above is to a full-resolution PDF!

Reply:
Added a high resolution PNG, and linking out more clearly to the original now. Thanks!